exportplan/ajax.py

```
@@ -36,6 +36,34 @@ def get(self, request):
         return JsonResponse(data)
 
 
+class ExportPlanRemoveCountryDataView(views.APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        if not self.request.GET.get('country'):
+            return HttpResponse(status=400)
+        country = self.request.GET.get('country')
+        try:
+            # To make more efficient by removing get
+            export_plan = helpers.get_exportplan(sso_session_id=self.request.user.session_id)
+            target_markets_filtered = filter(lambda i: i['country'] != country, export_plan['target_markets'])
+            data = list(target_markets_filtered)
+            export_plan = helpers.update_exportplan(
+                sso_session_id=self.request.user.session_id,
+                id=export_plan['pk'],
+                data={'target_markets': data}
+            )
+        except ReadTimeout:
+            return HttpResponse(status=504)
```

this hides the error from devs. if we did not catch the error then it would also be logged in sentry. currently it returns 5xx to user but we're blind to it happening